README.rst

```diff
@@ -174,6 +174,18 @@ See `the contributing page <https://github.com/01org/cc-oci-runtime/blob/master/
 Building
 --------
 
+Acquiring
+~~~~~~~~~
+
+As ``cc-oci-runtime`` contains some golang code, the source tree needs to live
+in your ``$GOPATH`` in order to build. The easiest way to get the codebase into
+the correct place is with the command::
```

Comment:
Please do not use :: use instead .. code-block:: language
In this case, the line above ends with "with the command:"
The next line is blank and the following then becomes:
.. code-block:: console

Please do not forget to add a blank line before the command and to indent it.